__tests__/api.test.js

```diff
@@ -179,6 +179,14 @@ describe("/api/articles", () => {
 						});
 					});
 			});
+			it("404: should respond with a message of 'Not Found' if none of the articles' topics match the topic value given", () => {
+				return request(app)
+					.get("/api/articles?topic=notATopic")
+					.expect(404)
+					.then(({ body }) => {
+						expect(body.msg).toBe("Not Found");
+					});
+			});
```

Comment:
Please add another test to check if the topic exists but there are no articles for it yet